scripts/update_customdc_stable_tag.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reassigns the customdc_stable tag to the HEAD commit on the specified remote.
+
+set -e
+
+REMOTE_NAME=$1
```

Comment:
since people use "dc" "origin" "upstream" differently. Maybe prompt remote url here for confirm?

Reply:
Done.